scripts/map.js

```diff
@@ -0,0 +1,69 @@
+jQuery(() => {
```

Comment:
We should test to see if `jQuery` is the same as `$`. One less global to worry about.